app/create/create-event-form.tsx

```diff
@@ -53,6 +53,7 @@ export const CreateEventForm: React.FC = () => {
       password: "",
       gatekeepers: [],
       discoverable: true,
+      communityId: null,
```

Comment:
i see above default values is empty string, should it be empty string instead of null for community (not sure it matters tho)